Jellyfin.Plugin.SubtitleExtract/Providers/SubsMetadataProvider.cs

```diff
@@ -0,0 +1,96 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Jellyfin.Plugin.SubtitleExtract.Tools;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Entities.Movies;
+using MediaBrowser.Controller.Entities.TV;
+using MediaBrowser.Controller.Library;
+using MediaBrowser.Controller.MediaEncoding;
+using MediaBrowser.Controller.Providers;
+using Microsoft.Extensions.Logging;
+
+namespace Jellyfin.Plugin.SubtitleExtract.Providers
+{
+    internal class SubsMetadataProvider : ICustomMetadataProvider<Episode>,
```

Comment:
You'll have better luck if this class is public

Just an initial pass, I'll do a full review when I have more time

Reply:
As you guessed, after making the class public works like a charm.
Settings are applied correctly as well.